src/io/flutter/settings/FlutterSettings.java

```diff
@@ -32,8 +30,6 @@
   private static final String legacyTrackWidgetCreationKey = "io.flutter.trackWidgetCreation";
   private static final String disableTrackWidgetCreationKey = "io.flutter.disableTrackWidgetCreation";
   private static final String useFlutterLogView = "io.flutter.useLogView";
-  private static final String memoryProfilerKey = "io.flutter.memoryProfiler";
-  private static final String newBazelTestRunnerKey = "io.flutter.bazel.legacyTestBehavior";
```

Comment:
Something else that's outside of memory profiler?

Reply:
yup, this was an unused field (from code removed by a previous PR)